assignment-2/README.md

```diff
@@ -1,12 +1,12 @@
 # IT3212 - Assignment 2
 
-Written and developed by Haakon Tideman Kanter, Henrik Skog, Mattis Hembre, Max Gunhamn, Sebastian Sole, and Magnus Rødseth.
+Written and developed by Haakon Tideman Kanter, Henrik Skog, Mattis Czternasty Hembre, Max Gunhamn, Sebastian Sole, and Magnus Rødseth.
 
 ## 1. Implement the preprocessing
 
 We started by removing features, according to what was outlined in assignment 1. Next, we removed rows with a confidence threshold below `1.0`.
 
-Next, we preprocessed the textual data. We cleaned up the `keyword` column, ensuring that all keywords were lower case, and that all keywords were separated by a single space. For comparison, some keywords were on the format `airplane%20accident`, which should be `airplane accident`.
+Next, we preprocessed the textual data. We cleaned up the `keyword` column, ensuring that all keywords were lower case becuase lowercasing the keywords allows for case-insensitive matching and simplifies the analysis process. We also ensured that all keywords were separated by a single space. For comparison, some keywords were on the format `airplane%20accident`, which should be `airplane accident`.
```

Comment:
Maybe don't need to justify the preprocessing allikavel here, because it is in the first report and the final report will also have it then